qeb_thamos_advise.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# workflow-helpers
+# Copyright(C) 2020 Francesco Murdaca
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""This is script for Qeb-Hwt thamos advise workflow task."""
+
+
+import os
+import json
+import yaml
+import logging
+from pathlib import Path
+
+from thamos.lib import advise_here
+from thamos.config import config
+from thoth.python.exceptions import FileLoadError
+from trigger_finished_webhook import trigger_finished_webhook
+
+_LOGGER = logging.getLogger("qebhwt")
+
+_GITHUB_EVENT_TYPE = os.getenv("GITHUB_EVENT_TYPE")
+_GITHUB_CHECK_RUN_ID = os.getenv("GITHUB_CHECK_RUN_ID")
+_GITHUB_INSTALLATION_ID = os.getenv("GITHUB_INSTALLATION_ID")
+_GITHUB_BASE_REPO_URL = os.getenv("GITHUB_BASE_REPO_URL")
+_ORIGIN = os.getenv("ORIGIN")
```

Comment:
they are not all mandatory actually, it depends on the context. maybe I should split the configuration depending on the context, because having them all for all the different helpers we create. WDYT @fridex ?

Reply:
My point was to avoid passing `None` values as soon as possible through the system if they are not valid inputs.

Reply:
Right, thanks, I will update them :)